findAllBundlesFromRoot.js

```diff
@@ -11,11 +11,16 @@ module.exports = function gloob(dir, cb) {
     glob(path.resolve(dir, '**/*.properties'), iferr(cb, function (ents) {
```

Comment:
Sadness. Not quite time to kill that beast but soon. node 4 works so much better!